ExampleMod/Content/Tiles/Plants/ExampleSapling.cs

```diff
@@ -47,6 +48,7 @@ public class ExampleSapling : ModTile
 			TileID.Sets.TreeSapling[Type] = true;
 			TileID.Sets.CommonSapling[Type] = true;
 			TileID.Sets.SwaysInWindBasic[Type] = true;
+			TileMaterials.SetForTileId(Type, TileMaterials._materialsByName["Plant"]);
```

Comment:
Please add a comment on why this is needed since its a newish inclusion and isn't obvious why